css/.scss-lint.yml

```diff
@@ -219,13 +220,19 @@ linters:
     SpaceAfterVariableName:
         enabled: true
 
+    # Checks for space around operators on values.
+    # Configure: `one_space` or `no_space`
+    SpaceAroundOperator:
+        enabled: true
+        style: one_space
```

Comment:
Yes that's correct. 

Hmm... Good question, I usually don't add space. 

Maybe we should change to `no_space` ?